CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # Icinga 2 | (c) 2012 Icinga GmbH | GPLv2+
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.17.5)
```

Comment:
As this is an assertion, GHA will for sure tell whether it's ok.

Reply:
No, this is not just an assertion on the CMake version, it also changes the behavior of whether CMake tries to be compatible with older versions: https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html#policy-version

So for a proper review of this, one also needs to look at the changes in CMake policies (i.e. these compatibility flags) and how they affect Icinga 2 (given the number of failed CI jobs, they do affect Icinga 2).

Reply:
Chill. All I wanted to say is: ok for GHA = ok for me.